mockcache.py

```diff
@@ -348,23 +354,21 @@ def check_key(key, key_extra_len=0):
         Is an unicode string (Raises MockcachedStringEncodingError)
         Is None (Raises MockcachedKeyNoneError)
     """
-    import types
-    if type(key) == types.TupleType:
+    if type(key) == tuple:
         key = key[1]
     if not key:
-        raise Client.MockcachedKeyNoneError, ("Key is None")
+        raise Client.MockcachedKeyNoneError("Key is None")
     if isinstance(key, unicode):
```

Comment:
indeed, I forgot to remove this one
it's correct for python 2 thanks to the __future__ import unicode_literals